app/assets/stylesheets/components/_card_list.css.sass

```diff
@@ -26,6 +26,13 @@
       &.ng-enter-active
         animation-name: bounceInLeft
 
+    .CardList-card.is-closed:nth-child(#{$i})
```

Comment:
This doesn't need to be within the ```for``` loop neither inside the 
```sass
.CardList.CardList--withAnimation
```
The animation will work independently of the state.

Also you don't need the ```nth-child```, you are applying the same style to every child right?